app/models/email_address.rb

```diff
@@ -42,12 +42,7 @@ def fed_or_mil_email?
   end
 
   def fed_email?
-    if IdentityConfig.store.use_fed_domain_class
-      return false unless domain
```

Comment:
I would have expected we'd retain this check to return `false` unless there's a domain. Is there a re ason we removed it?

Do we have test coverage for that behavior?

Reply:
You are correct. I don't think there was test coverage for that behavior. Will add it.

Reply:
Correction. `ActiveRecord#exists?` handles the situation correctly. I'll just 
add a test.